pkg/utils/utils.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/openshift/origin/third_party/github.com/docker/libcompose/project"
 )
 
+// CheckIfFileExists checks if all files in array exists and that they are files (not directories).
+func CheckIfFileExists(files []string) error {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			return fmt.Errorf("main: file %q not found", filename)
```

Comment:
> And my question is why would you add "main: "? 
> I guess this is because it comes from main function, right?
Yep right

> Why should user see that? He doesn't care about that.
Actually, I saw that in some places and I think it is a best practice? Not sure, I am ok with whatever is standard way of doing things.